delivery_service.rb

```diff
@@ -0,0 +1,37 @@
+class DeliveryService 
```

Comment:
Due to specification the service should store pool of transport and provide functionality for finding available one, delivery confirmation . 

```
Парк службы доставки имеет несколько велосипедов и машин(количество определяете сами). Создать 2 метода для этого класса. Первый по параметрам доставки(вес и дистанция) находит подходящий транспорт для доставки. Второй метод подтверждает доставку. 
```
So, define pool, method for selecting transport from pool by availability criteria and some small confirmation method (availability toggling and some message for example)

Reply:
Pool added